evaporate.js

```diff
@@ -1236,14 +1236,22 @@
                     (request.contentType || '') + '\n' +
                     '\n' +
                     x_amz_headers +
-                    (con.cloudfront ? '/' + con.bucket : '') +
+                    (prepend_bucket ? '/' + con.bucket : '') +
                     request.path;
                 return to_sign;
             }
 
+            function isCloudfront() {
+              return con.cloudfront || AWS_URL.indexOf('cloudfront') > -1;
+            }
+
+            function isAcceleratedS3() {
+              return AWS_URL.indexOf('.s3-accelerate.amazonaws.com') > -1;
+            }
+
             function getPath() {
                 var path = '/' + con.bucket + '/' + me.name;
-                if (con.cloudfront || AWS_URL.indexOf('cloudfront') > -1) {
+                if (isCloudfront() || isAcceleratedS3()) {
```

Comment:
These helper methods are not defined in this PR and do not exist in the committed code.